src/options/normalizeOptions.js

```diff
@@ -203,6 +201,33 @@ function normalizeProjection (options) {
   else return `EPSG:${projection}`
 }
 
+/**
+ * Ensure idField is set if metadata doesn't have a value but a field named OBJECTID is present
+ * @param {object} metadata
+ */
+function normalizeIdField (options, features = []) {
+  const collection = options.collection || {}
+  const metadata = collection.metadata || {}
+  const feature = features[0] || {}
+  const featureProperties = feature.properties || feature.attributes || {}
+  let idField = null
+
+  // First, check metadata for idField
+  if (metadata.idField) idField = metadata.idField
+
+  // Check metadata.fields for and OBJECTID property
+  else if (_.find(metadata.fields, { name: 'OBJECTID' })) idField = 'OBJECTID'
+  // Check features for an OBJECTID property
+  else if (features.length > 0 && featureProperties.OBJECTID) idField = 'OBJECTID'
```

Comment:
Very good catch!  I've changed this in https://github.com/koopjs/winnow/pull/104/commits/fd9f8924c6c6f83ee210d962ca88b13f80b4d4ef so that it explicitly checks for defined, non-null OBJECTID.